### PR TITLE
Added ds-for-beginners.gif

### PR DESCRIPTION
**Pull request by mohitjaisal**
- Added ds-for-beginners.gif to readme .md 
- The gif is fully compressed and load very fast

## Before:
image

## After:
[Promo video](https://youtu.be/8mzavjQSMM4 "Promo video")

### CODE REVIEW

**Comment by jlooper** [comment]
oh wow! @mohitjaisal is there any way you could do this for our other curricula (shameless steal here, Jasmine! 🥇 ) - aka.ms/iot-beginners, aka.ms/ml-beginners, aka.ms/webdev-beginners

**Comment by mohitjaisal** [comment]
> oh wow! @mohitjaisal is there any way you could do this for our other curricula (shameless steal here, Jasmine! 🥇 ) - aka.ms/iot-beginners, aka.ms/ml-beginners, aka.ms/webdev-beginners

Of course @jlooper . It will be my pleasure to work on these repositories ✌️

**Comment by mohitjaisal** [comment]
> @mohitjaisal wow!!! I love this, thank you 😍. My final ask is that you credit yourself. I've added a comment on where to place this

 Thanks for the credit @paladique. I am glad that you liked it. 😋

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Azure Cloud Advocates at Microsoft are pleased to offer a 10-week, 20-lesson cur
 
 ## Meet the Team
 
-[![Promo video](ds-for-beginners.png)](https://youtu.be/8mzavjQSMM4 "Promo video")
+[![Promo video](ds-for-beginners.gif)](https://youtu.be/8mzavjQSMM4 "Promo video")
 
 > 🎥 Click the image above for a video about the project  the folks who created it!
```
[the remark this answers is not in the document]
Comment (mohitjaisal) [reply README.md:37]: @paladique [Added the credit](https://github.com/microsoft/Data-Science-For-Beginners/pull/221/commits/8aaa0c7891b63e29d1c237d1f845ed198fdf40d1) . Thank you so much!!

**Pull request merged**